conf/machine/include/tegra-common.inc

```diff
@@ -23,7 +23,7 @@ TNSPEC ?= "${TEGRA_BOARDID}-${TEGRA_FAB}-${TEGRA_BOARDSKU}-${TEGRA_BOARDREV}-1-$
 
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-tegra"
 PREFERRED_PROVIDER_virtual/bootloader ?= "tegra-uefi-prebuilt"
-PREFERRED_PROVIDER_virtual/secure-os ?= "tos-prebuilt"
+PREFERRED_PROVIDER_virtual/secure-os ?= "tos-tegra"
```

Comment:
Since the OPTEE tests do not currently pass with the from-source build, we should stick to the prebuilt copy by default.

Reply:
Okay, will do that